src/hooks/useBridge.ts

```diff
@@ -75,14 +69,12 @@ export const useBridge = () => {
   }
 
   const getPIQBalance = () => {
-    if (pTokenBalance) return utils.formatEther(pTokenBalance)
-
+    if (pTokenBalance) return ethers.utils.formatEther(pTokenBalance.value)
```

Comment:
why import ethers if we already imported utils ?

Reply:
I will remove that now.

Reply:
@kesar, I remember I had to rework it because the UI was weird.

<img width="910" alt="image" src="https://user-images.githubusercontent.com/70170061/205700629-18aee66a-404c-4f28-9dbd-e665ecaef30e.png">